tests/unit/io/local/test_local.py

```diff
@@ -6,10 +6,61 @@
 import pandas as pd
 import pytest
 
-from sdv.io.local.local import CSVHandler, ExcelHandler
+from sdv.io.local.local import BaseLocalHandler, CSVHandler, ExcelHandler
 from sdv.metadata.multi_table import MultiTableMetadata
 
 
+class TestBaseLocalHandler:
+
+    def test___init__(self):
+        """Test the dafault initialization of the class."""
+        # Run
+        instance = BaseLocalHandler()
+
+        # Assert
+        assert instance.decimal == '.'
+        assert instance.float_format is None
+
+    def test_create_metadata(self):
+        """Test that ``create_metadata`` will infer the metadata."""
+        # Setup
+        data = {
+            'hotel': pd.DataFrame({
+                'hotel_id': [1, 2, 3, 4, 5],
+                'stars': [3, 4, 5, 3, 4]
+            }),
+            'guests': pd.DataFrame({
+                'guest_id': [1, 2, 3, 4, 5],
+                'hotel_id': [1, 1, 3, 2, 3]
+            })
+        }
+        instance = BaseLocalHandler()
+
+        # Run
+        metadata = instance.create_metadata(data)
+
+        # Assert
+        assert isinstance(metadata, MultiTableMetadata) is True
```

Comment:
You don't need `is True` here I think

Reply:
I think that I started doing this after `assert not False` or `assert None` started asking me to do `assert is None` or `assert is False`. But I think its fine to be just `assert`.